src/components/New Calendar/Calendar.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
```

Comment:
This new calendar folder is used just to create datepicker because the old one was too tightly coupled with the carousel. I'll pbly come back and try to merge this calendar with the old one and make it compatible with both carousel and datepicker but for now a couple of these files are duplicates